package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azurefunctions",
     "displayName": "Azure Functions",
     "description": "%azureFunctions.description%",
-    "version": "1.7.4-alpha.0",
+    "version": "1.7.4-alpha.1",
```

Comment:
@nturinski this would be needed in package-lock.json as well.

Reply:
Damn it, I didn't want to actually pull this down and do it for him lol.

Reply:
Lol I can do it if you tell me what it's about

Reply:
Interesting - for the record I think you could automate that in the dev ops pipeline. I assume the purpose is to differentiate prerelease builds? If you chop off the ".0" from the version in git and just add it every build from main I think you'd get the same thing with less manual steps and less git churn. For example add something like this to your azure pipelines:

```
CURR_VER=$(node -p "require('./package.json').version")
npm version "$CURR_VER.$TODO"
```

Where TODO is either a build id or [counter](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#counter) depending on your preferences

Reply:
The reason we started bumping the versions like this is if you've installed an extension from a VSIX, VS Code won't let you install (update) the same extension from a VSIX unless the extension version is different. 

I think your solution would do the trick as well. I didn't know about counter, that's cool.

Reply:
Anyways I bumped the lock file for this PR. I see tests are failing, but looks like tests have been failing for a while on main so hard to tell if my PR caused that. Let me know if you want me to investigate

Reply:
> 

I could check again but I thought I hadn't experienced that issue. I thought the problem was that the publicly-released version of the extension had a version number higher than what we were using internally for the unified work, and VSCode kept auto-updating it.

Reply:
> > The reason we started bumping the versions like this is if you've installed an extension from a VSIX, VS Code won't let you install (update) the same extension from a VSIX unless the extension version is different.
> > I think your solution would do the trick as well. I didn't know about counter, that's cool.
> 
> I could check again but I thought I hadn't experienced that issue. I thought the problem was that the publicly-released version of the extension had a version number higher than what we were using internally for the unified work, and VSCode kept auto-updating it.

I think you're right. I can't reproduce the issue I described either. For some reason I thought for sure that was an issue we had. 

If the issue you described is the only reason we need these version bump checks, then we can probably remove them...